CHANGELOG.md

```diff
@@ -3,13 +3,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [2.0.0] - TBD
+## [2.0.0] - 2019-05-23
 
 ### Added
 - Option to configure S3 storage class for cost optimization.
 - Change in structure of `apiary_managed_schemas` variable from list to list of maps.
 
-## [1.1.0] - TBD
+## [1.1.0] - 2019-05-23
```

Comment:
this looks weird is this what @mroark1m was talking about that should be merged with 2.0.0? Are we actually going to have a 1.1.0 release?